.github/workflows/deploy-alerts-prod.yaml

```diff
@@ -16,4 +16,6 @@ jobs:
         env:
           APIKEY: ${{ secrets.NAIS_DEPLOY_APIKEY }}
           CLUSTER: prod-gcp
-          RESOURCE: nais/alerts.yaml
\ No newline at end of file
+          RESOURCE: nais/alerts.yaml
+          # Nødvendig for å trigge templating engine på alerts.yaml
+          VAR:
```

Comment:
`VAR: isDev=false `?

Reply:
Jo, vi kan ha det, men jeg ville prøve å ha så lite config i prod deploy som mulig. Helst skulle jeg ikke hatt den `VAR` biten i det hele tatt, men den var visst nødvendig for at Handlebars enginen skal blir kjørt.

Føler at om vi legge til den variabelen så øker vi muligheten for at den på ett tidspunkt blir truthy med ett uhell.